### DISCUSSION TITLE
Increasing arrows click area

### DISCUSSION THREAD

**Discussion by R00GER** [Eliav2/react-xarrows#59] [Q&A]
Hi,

Can you figure out some way to increase the area for clicking the arrows without increasing the strokeWidth? The strokeWidth of the arrows must be 2px and it's difficult to hit those.

Currently as a workaround I'm handling this by drawing two lines, visible and thicker invisible line.

I would appreciate any suggestions.

Thanks!

**Comment by R00GER** [comment]
@Eliav2

**Reply by Eliav2** [reply]
Im unsure but maybe its possible. You will have to pass `pointerEvents` by using passBodyProps and passHeadProps(you will probably need to pass diffrent value)

If this did not work for Here’s an idea,insead of using one xarrow use 2 xarrows with the same props. On the first xarrows use passProps={pointerEvents =none} 
and on the later rendered xarrow increace strokeWidth and pass opacity 0

 Good luck, let me know if it works
